build.gradle

```diff
@@ -142,6 +142,8 @@ dependencies {
 
     includeJ8("com.viaversion:viaversion:${rootProject.viaver_version}")
     include("com.github.TinfoilMC:ClientCommands:1.1.0")
+
+    testImplementation("org.testng:testng:6.13.1")
```

Comment:
Why is TestNG not on 7.10.2?

Reply:
Gradle refused to resolve it, and that was the version IntelliJ 2022.2.2 suggested.

I can try again and see if it'll accept it later.